client/src/components/DataFiles/DataFilesTable/DataFilesTable.scss

```diff
@@ -33,8 +33,13 @@
   padding-right: 15px;
```

Comment:
__Tweaked, But Postponing__
The column width off-ness will still happen on narrow and medium screens. If you watch when the table has a vertical scrollbar and make screen wider or narrower the cell content and cell header will align and misalign as you go. This doesn't happen with Allocations or Dashbaord (which uses real table elements)—whatever offness they have is consistent. But, like most Data Files table UI, the real solution is converting it to use InfiniteScrollTable ([FP-818](https://jira.tacc.utexas.edu/browse/FP-818)) or making DataFiles use proper table markup instead of divs.

I have:
1. 6aaab34 resolved the "double padding" for the "View Path" link ([Cell Pad @ __Wide Section__ + Small Text](https://user-images.githubusercontent.com/62723358/116605089-975eef80-a8f4-11eb-8f10-32a5655783f0.png))
2. bb66e2b reduced the font-size of Data Files tables to match other sections' tables (see screenshots of other items)
3. bb66e2b updated the min-width for Data Files based on the smaller font size ([Scrolling @ <960](https://user-images.githubusercontent.com/62723358/116604911-641c6080-a8f4-11eb-8eee-2bb411c97e2e.png)) ([No Scrolling @ =960](https://user-images.githubusercontent.com/62723358/116604916-64b4f700-a8f4-11eb-82e6-ff56dfd2288d.png))

Reply:
__Tweaked, But Postponed__
Man… I wrote a real good comment about this. I can't find the comment.

_Short Version_: Data Files needs to use real table markup, not `<div>`s—whether it uses `<InfiniteScrollTable>` ([](https://jira.tacc.utexas.edu/browse/FP-818)) or just updates its markup, I don't have a professional opinion.

Reply:
__Update__

Merge of #401 may have helped, also. Still, Data Files should use `<table>`'s, first.